### ISSUE TITLE
magnetizer doesn't have a crafting recipe

### ISSUE THREAD

**Issue by Toastuun** [omeranha/MekaWeapons#2]
The logs say that there is an unknown item of "forge:ingots/steel". Briefly looking at the src, the recipe tries to use "forge:ingots/steel" as an item, and not a tag. Other recipes do use it as a tag though. I'm assuming that that's the issue, but i'm not great at recipe coding.

**Comment by FishingHacks** [comment]
yep, that seems to be the issue, i could make a pull request but im too lazy, i hope the author fixes it soon.
[fix_mekaweapons_recipes.zip](https://github.com/omeranha/MekaWeapons/files/13448417/fix_mekaweapons_recipes.zip)
this datapack should fix it

**Comment by omeranha** [comment]
sorry for the delay guys, just a small issue that was passed by, thank you!
